src/main/java/com/avanza/gs/test/PartitionedPu.java

```diff
@@ -53,7 +53,7 @@ public PartitionedPu(PartitionedPuConfigurer configurer) {
 		this.numberOfBackups = configurer.numberOfBackups;
 		this.numberOfPrimaries = configurer.numberOfPrimaries;
 		this.contextProperties.putAll(configurer.contextProperties);
-		this.beanProperies.putAll(configurer.beanProperies);
+		this.beanProperies.putAll(configurer.beanProperties);
```

Comment:
Maybe we can fix the spelling of `this.beanProperies` as well? 😃

Reply:
Well spotted. Fixed in a separate commit.